frontend/src/pages/Home.jsx

```diff
@@ -1,5 +1,7 @@
 function Home() {
-  return <p>Home</p>;
+  return (
+    <img src="./src/assets/logo1.png" className="logohome" alt="logohome" />
```

Comment:
choose a better alt: avoid the words logo, image, picture...

some guidelines: https://www.w3.org/WAI/tutorials/images/